core/toolbox/category.ts

```diff
@@ -138,7 +138,7 @@ export class ToolboxCategory
       'label': 'blocklyTreeLabel',
       'contents': 'blocklyToolboxCategoryGroup',
       'selected': 'blocklyTreeSelected',
-      'openicon': 'blocklyToolboxCategoryIconOpen',
+      'openicon': 'blocklyTreeIconOpen',
       'closedicon': 'blocklyTreeIconClosed',
```

Comment:
Hello, it looks like this might have gotten messed up in your rebase? These should both be changed to use `blocklyToolboxCategory...` instead of `blocklyTree...`